### ISSUE TITLE
RemoteTech integration doesn't load (CC 1.10.1 KSP1.1.0.1224)

### ISSUE THREAD

**Issue by cherrydev** [jrossignol/ContractConfigurator#502]
Earlier I get:
```
[LOG 23:57:11.461] Load(Assembly): ContractConfigurator/CC_RemoteTech
[LOG 23:57:11.461] AssemblyLoader: Loading assembly at <foo>/Kerbal Space Program/GameData/ContractConfigurator/CC_RemoteTech.dll
[LOG 23:57:11.463] AssemblyLoader: KSPAssembly 'CC_RemoteTech' V1.0
[LOG 23:57:11.463] AssemblyLoader: KSPAssemblyDependency 'RemoteTech' V1.6
[LOG 23:57:11.463] AssemblyLoader: KSPAssemblyDependency 'ContractConfigurator' V1.0
[LOG 23:57:11.463] Load(Assembly): ContractConfigurator/ContractConfigurator
```

But then later:

```
[LOG 23:57:11.524] AssemblyLoader: Loading assemblies
[ERR 23:57:11.569] AssemblyLoader: Exception loading 'CC_RemoteTech': System.Reflection.ReflectionTypeLoadException: The classes in the module cannot be loaded.
  at (wrapper managed-to-native) System.Reflection.Assembly:GetTypes (bool)
  at System.Reflection.Assembly.GetTypes () [0x00000] in <filename unknown>:0 
  at AssemblyLoader.LoadAssemblies () [0x00000] in <filename unknown>:0 

Additional information about this exception:

 System.TypeLoadException: Could not load type 'ContractConfigurator.RemoteTech.CelestialBodyCoverageFactory' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type 'ContractConfigurator.RemoteTech.CelestialBodyCoverageParameter' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type 'ContractConfigurator.RemoteTech.ActiveVesselRangeRequirement' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type 'ContractConfigurator.RemoteTech.CelestialBodyCoverageRequirement' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type 'ContractConfigurator.RemoteTech.HasAntennaParameter' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type 'ContractConfigurator.RemoteTech.HasAntennaFactory' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type 'ContractConfigurator.RemoteTech.RemoteTechProgressTracker' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type 'ContractConfigurator.RemoteTech.RemoteTechProgressTracker+FakeSatellite' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type 'ContractConfigurator.RemoteTech.RemoteTechAssistant' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type 'ContractConfigurator.RemoteTech.VesselConnectivityFactory' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type 'ContractConfigurator.RemoteTech.VesselConnectivityParameter' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type 'ContractConfigurator.RemoteTech.SignalDelayFactory' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type 'ContractConfigurator.RemoteTech.SignalDelayParameter' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type 'ContractConfigurator.RemoteTech.KSCConnectivityParameter' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type 'ContractConfigurator.RemoteTech.KSCConnectivityFactory' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type '<>c__DisplayClassf' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type '<get_Antennas>d__2' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.

 System.TypeLoadException: Could not load type '<FindNeighbors>d__5' from assembly 'CC_RemoteTech, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
```

I have RemoteTech v[1.6.11](https://github.com/RemoteTechnologiesGroup/RemoteTech/releases/tag/1.6.11)

**Comment by cherrydev** [comment]
I'm actually a bit confused about what version of RemoteTech you linked this against.  1.10.1 was released on March 31st or April 1st, but the first KSP 1.1 version of RemoteTech that was released (1.6.10) was only released on the April 12th.  What version of RemoteTech were you building against?

**Comment by cherrydev** [comment]
I just re-compiled it against RT v1.6.11 (the latest release) and it appears to be working now, though I haven't attempted to accomplish any of the missions.

**Comment by jrossignol** [comment]
Yeah, the old CC_RemoteTech.dll was included by mistake in old builds prior to when RemoteTech was available for 1.1.  I haven't had a chance to recompile since then.  Will be fixed in the next release.

**Comment by jrossignol** [comment]
Fixed for 1.10.2
